Transformer/LuceneToBddTransformer.php

```diff
@@ -0,0 +1,183 @@
+<?php
+
+namespace OpenOrchestra\Transformer;
+
+use OpenOrchestra\Transformer\LuceneToBddTransformerInterface;
+use OpenOrchestra\Backoffice\Exception\MissingFieldTransformerException;
```

Comment:
orchestra mondo libs hasn't dependency to backofficeBundle, you can't use this exception